packages/web-components/lib/components/selfie/src/selfie-capture-wrapper/SelfieCaptureWrapper.tsx

```diff
@@ -29,16 +29,33 @@ const SelfieCaptureWrapper: FunctionComponent<Props> = ({
   hidden: hiddenProp = false,
   ...props
 }) => {
+  const isParentCypress = (() => {
+    try {
+      return (
+        !!(window.parent as any).Cypress ||
+        (window.parent.navigator.userAgent.includes('Electron') &&
+          (window.parent as any).__Cypress)
+      );
+    } catch {
+      return false;
+    }
+  })();
+  const isCypress =
+    isParentCypress ||
+    !!(window as any).Cypress ||
+    (window.navigator.userAgent.includes('Electron') &&
+      (window as any).__Cypress);
```

Comment:
once this go in we should rethink this

Reply:
Yeah ideally we would want to also test the face detection component. It may be possible to spoof the camera feed with a video of a user, but i havent been able to dig down that rabbit hole yet.